style/sass/README.md

```diff
@@ -51,6 +51,26 @@
 * Avoid using HTML tags on classes with specific class names like `.featured-articles`.
 * Avoid using comma delimited selectors.
 * Avoid nesting within a media query.
+* Don't concatenate selectors using Sass's parent selector (`&`):
+    ```scss
```

Comment:
Yeah, probably. As you can see, I'm playing around a bit with our guides layout 😉